lib/components/UploadPicker.vue

```diff
@@ -240,54 +300,116 @@ export default Vue.extend({
 	methods: {
 		/**
 		 * Trigger file picker
+		 * @param folder Upload folders
 		 */
-		onClick() {
-			this.$refs.input.click()
+		onTriggerPick(folder = false) {
+			const input = this.$refs.input as HTMLInputElement
+			// Setup directory picking if enabled
+			if (this.canUploadFolders) {
+				input.webkitdirectory = !!folder
```

Comment:
Maybe more readable if this is what you cant to achieve?

```suggestion
				input.webkitdirectory = folder !== false
```